app/views/hyrax/dashboard/collections/_form_relationships.html.erb

```diff
@@ -4,37 +4,28 @@
   <section>
     <h3><%= t('.headlines.this_collection_in_other_collections') %></h3>
     <p><%= t('.this_collection_in_other_collections_description') %></p>
-
-    <article class="form-inline select-wrapper">
-      <div class="form-group">
-        <label for="exampleInputName2"><strong><%= t('hyrax.dashboard.my.action.add_to_collection') %>:</strong></label>
-        <select class="form-control" id="add_to_collection_selection">
-          <option><%= t('.select_a_collection') %></option>
-        </select>
-      </div>
-      <button class="btn btn-primary"><%= t('helpers.action.add') %></button>
-    </article>
-
     <article class="collection-list-wrapper">
+      <% if @form.available_parent_collections.any? %>
       <p><strong><%= t('.collection_is_subcollection_description') %></strong></p>
       <table class="table table-striped table-condensed">
         <thead>
           <tr>
             <th><%= t('.table.collection_title') %></th>
-            <th><%= t('.table.action') %></th>
           </tr>
         </thead>
         <tbody>
-          <tr>
-            <td><a href="#">Butterfiles: Our Favorites!</a></td>
-            <td><button class="btn btn-sm btn-danger remove-from-collection-button"><%= t('.buttons.remove_from_collection') %></button></td>
-          </tr>
-          <tr>
-            <td><a href="#">Recent Published Research Publications and Data On Macrolepidopteran Clade</a></td>
-            <td><button class="btn btn-sm btn-danger remove-from-collection-button"><%= t('.buttons.remove_from_collection') %></button></td>
+          <% @form.available_parent_collections.each do |document| %>
+            <tr>
+              <td>
+                <% id = document.id %>
```

Comment:
Can this be combined with the next line as it’s only used once

Reply:
Yup. I think I just copied that code from another view because I was trying to make a consistent way of creating the links. But that's an easy change.